hal/armv7a/imx6ull/imx6ull.c

```diff
@@ -360,5 +382,12 @@ void imx6ull_init(void)
 	/* Enable EPITs clocks */
 	*(imx6ull_common.ccm + ccm_ccgr1) |= 0x00005000;
 
+	/* Save SRSR */
+	imx6ull_common.resetFlags = (*(imx6ull_common.ccm + src_srsr) & 0x101fdu);
+
+	/* Clear SRSR */
+	*(imx6ull_common.src + src_srsr) = 0xfdu;
```

Comment:
Please remove.

The OS needs to read boot reason during / after bootup - **this breaks DTR project**.

Please note that this functionality probably breaks reading reboot reason for other platforms (see the implementation of `pctl_reboot` with action `pctl_get` for various platforms).